cupy/core/elementwise.pxi

```diff
@@ -97,13 +97,19 @@ cdef dict _python_type_to_numpy_type = {
     bool: numpy.dtype(bool).type,
 }
 
+_int_iinfo = numpy.iinfo(int)
```

Comment:
Please use `cdef long long _int_min` , `cdef long long _int_max` and `cdef _int_type` to up performance.

Reply:
Thanks, fixed.